.github/workflows/quickstarts.eu-production.yml

```diff
@@ -0,0 +1,19 @@
+name: Submit All - EU-Production
```

Comment:
im not sure why these are showing up as new files when i pulled it from main... we can probably ignore these workflow files